tools/releaseBuild/macOS/PowerShellPackageVsts.ps1

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# PowerShell Script to build and package PowerShell from specified form and branch
+# Script is intented to use in Docker containers
+# Ensure PowerShell is available in the provided image
+
+param (
+    # Set default location to where VSTS cloned the repository locally.
+    [string] $location = $env:BUILD_REPOSITORY_LOCALPATH,
+
+    # Destination location of the package on docker host
+    [Parameter(Mandatory,ParameterSetName='Build')]
+    [string] $destination = '/mnt',
+
+    [Parameter(Mandatory,ParameterSetName='Build')]
+    [ValidatePattern("^v\d+\.\d+\.\d+(-\w+(\.\d+)?)?$")]
+    [ValidateNotNullOrEmpty()]
+    [string]$ReleaseTag,
+
+    [Parameter(ParameterSetName='Build')]
+    [ValidateSet("zip", "tar")]
+    [string[]]$ExtraPackage,
+
+    [Parameter(Mandatory,ParameterSetName='Bootstrap')]
+    [switch] $BootStrap,
+
+    [Parameter(Mandatory,ParameterSetName='Build')]
+    [switch] $Build
+)
+
+# We must build in /PowerShell
+$repoRoot = '/PowerShell'
+if($BootStrap.IsPresent)
+{
+    $repoRoot = $location
+}
+
+if($Build.IsPresent)
+{
+    # cleanup the folder but don't delete it or the build agent will loose ownership of the folder
+    Write-Verbose -Message "cleaning /PowerShell" -Verbose
+    Get-ChildItem -Path /PowerShell/* -Attributes Hidden,Normal,Directory | Remove-Item -Recurse -Force
+
+    dir env:
```

Comment:
I guess this is for diagnostics purpose?

Reply:
I will remove